moon/controller_round1.py

```diff
@@ -133,29 +133,37 @@ def on_object_reached(self, data):
         ):
             x,y,z = self.xyz
             print(self.sim_time, "app: Object %s[%d] reached at [%.1f,%.1f], queueing up 19 around" % (object_type, object_index, x, y))
-            heapq.heappush(self.volatile_queue, (1, [object_index, object_type, x, y]))
+            heapq.heappush(self.volatile_queue, (self.sim_time.total_seconds(), [object_index, object_type, x, y]))
             for i in range(6):
                 x_d,y_d = pol2cart(3.46, math.radians(i * 60))
-                heapq.heappush(self.volatile_queue, (2, [object_index, object_type, x+x_d, y+y_d]))
+                heapq.heappush(self.volatile_queue, (3600 + self.sim_time.total_seconds(), [object_index, object_type, x+x_d, y+y_d]))
                  # https://www.researchgate.net/figure/Minimum-overlap-of-circles-covering-an-area_fig2_256607366
                  # Minimum overlap of circles covering an area; r= 3.46 (=2* r * cos(30)); step 60 degrees, 6x
             for i in range(12):
                 x_d,y_d = pol2cart(2*3.46, math.radians(30 + i * 60))
```

Comment:
this is wrong, right?
1) i*60 means that you report the same location twice and ignore every second
2) offset 30 should be zero (at least looking at the picture where all 3 circles are on one line) but the result could be the same ... I would almost double check and draw all 19 reporting circles :)

Reply:
Funny enough I did draw it but I see what you mean. Maybe I messed it up subsequently. I will look into this, thanks. Bad news is that the second circle of guesses rarely gets used, ie the localization is rarely off by 4-6m, either it's pretty accurate or it thinks it's somewhere else on the map where the difference from true pose is 10m+ or it declares itself lost altogether.

Reply:
Visually I am thinking this is the correct solution and the formula for that is substantially different than the existing code, I guess I was a little too quick coding here
![Figure_1](https://user-images.githubusercontent.com/11981914/91471955-c18f7780-e864-11ea-8899-40b02bf7d176.png)

Reply:
the latest commit looks OK, thanks m.